reframe/frontend/statistics.py

```diff
@@ -126,7 +126,7 @@ def failure_stats(self):
             partname = partition.fullname if partition else 'None'
             environ_name = (check.current_environ.name
                             if check.current_environ else 'None')
-            f = "[%s, %s, %s]" % (check.name, environ_name, partname)
+            f = f'[{check.name}, {check.environ_name}, {check.partname}]'
```

Comment:
This is why the unit tests are failing:

```suggestion
            f = f'[{check.name}, {environ_name}, {partname}]'
```

Reply:
It is weird. My unit tests are passed. I think I should add full list check in the unittest instead of check name only.

Reply:
Indeed strange. I've fixed that.